plop/general/component-stories-helpers-js.txt

```diff
@@ -0,0 +1,11 @@
+import { Link, Tip } from "vibe-storybook-components";
+
+export const TipOtherComponent = () => (
```

Comment:
Why is this a "helper" and not a component in `storybook-vibe-components`?

Reply:
No, it's different
This is just a `Tip` which you render in a mdx file with a link to some other component
I'd render it normally in the mdx without any helpers as we did before, but starting from storybook 7, we have an issue with that

I'll give an example, just so it'd be easier. Let's say we have a tip like this
```
  <Tip title="Use color contrast tools for code and design">
    <div>
      To check color contrast in design use
      <Link href="https://www.figma.com/community/plugin/748533339900865323/Contrast" size={Link.sizes.SMALL}>
        contrast figma plugin
      </Link>
      and inspect elements accessibility tab for live code verification
    </div>
  </Tip>
```
If you'd render it in mdx it'll look like this
<img width="912" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/104433616/0f0e841c-6dde-4248-b596-1fa84307e120">
Cause this stuff is treating text around the `Link` like `p` and it's causing this

You could have solved it by wrapping all the text in `span` but I thought it's not really convenient.
Besides it's not the only use case when mdx is playing tricks with you, so that's why I found this `.stories.helpers.js` workaround